sdk/build.gradle.kts

```diff
@@ -0,0 +1,124 @@
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    kotlin("jvm")
+    `java-library`
+    `maven-publish`
+    id("org.jetbrains.dokka")
+    signing
+}
+
+group = "org.virtuslab"
+version = "0.1.0-SNAPSHOT"
```

Comment:
Not sure what version we want to release as our first one. `0.1.0` or `1.0.0`?

Reply:
I would go with `0.1.0`

Reply:
Unless... maybe we should follow the same pattern as with provider SDKs and name it after Pulumi's version? In this case `0.9.4.0`? Otherwise we stumble upon the same problem when it comes to updating the code vs updating the dependency.

Reply:
I didn't think much about it, but yes, seems to be a better choice!